### PR TITLE
Add Browser component

### PR DESCRIPTION
**Pull request by katydecorah**
image

We can wrap it around any type of media we want to appear in a browser:

```html
<Browser>
  <img src="cool-cats-skateboarding.jpg alt="" />
</Browser>
```

Ref #147 

🐌 no rush

### CODE REVIEW

File: src/components/browser/__tests__/browser-test-cases.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+const testCases = {};
+
+testCases.basic = {
+  component: Browser,
+  description: '',
+  props: {
+    children: (
+      <img
+        src="https://docs.mapbox.com/studio-manual/assets/ref-styles-manage-styles-1200-599fd585f5f2d8e0729d5fe37f9722c0.webp"
```
Comment (colleenmcginnis) [inline src/components/browser/__tests__/browser-test-cases.js]: This image is not loading for me in Safari.

image

Comment (katydecorah) [reply src/components/browser/__tests__/browser-test-cases.js]: Ohh is that the example image is broken? I'm using a webp image, I should use a png instead!

Comment (katydecorah) [reply src/components/browser/__tests__/browser-test-cases.js]: image

Fixed! I added an `assets` folder in `src/tests-cases-app/` so we can store images/videos/etc that will be bundled only for the test cases site.

Comment (colleenmcginnis) [reply src/components/browser/__tests__/browser-test-cases.js]: oooo yes!

**Pull request merged**